src/main/java/com/vacation_bot/domain/models/VacationRequestBody.java

```diff
@@ -0,0 +1,14 @@
+package com.vacation_bot.domain.models;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+public class VacationRequestBody {
+    private String userName;
+    private String startDate;
+    private String endDate;
```

Comment:
Write java doc for each field and specify `@JsonProperty` as in the task description.